.github/workflows/deploy_api.yaml

```diff
@@ -1,30 +1,24 @@
 name: Deploy API App Function
-
 on:
-  workflow_dispatch:
+  workflow_dispatch: null
```

Comment:
why null?

Reply:
The source file `deploy_api.yaml` contained `workflow_dispatch:` (with an empty value)

The `null` value is  the result of `YAML.parse -> zod.parse -> YAML.stringify`, but I don't know exactly which step of those caused the change. Anyway, I think that is not a regression because an empty value is equivalent to null in YAML.

If it was caused by `zod` I think we can remove this library by parsing/validating the `yaml` AST direclty.

Reply:
@gunzip I improved this part [switching to AST manipulation](https://github.com/pagopa/dx/pull/922), now we change only the affected properties